DPGAnalysis/MuonSysAging/plugins/GEMChamberMasker.cc

```diff
@@ -0,0 +1,188 @@
+// system include files
+#include <memory>
+#include <string>
+#include <vector>
+#include <iostream>
+#include <algorithm>
+#include <regex>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDProducer.h"
+#include "FWCore/Framework/interface/ConsumesCollector.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DataFormats/Common/interface/DetSetVector.h"
+#include "DataFormats/GEMDigi/interface/GEMDigiCollection.h"
+#include "DataFormats/GEMDigi/interface/GEMPadDigiCollection.h"
+#include "DataFormats/GEMDigi/interface/GEMCoPadDigiCollection.h"
+#include "DataFormats/MuonDetId/interface/GEMDetId.h"
+#include "CondFormats/MuonSystemAging/interface/MuonSystemAging.h"
+#include "CondFormats/DataRecord/interface/MuonSystemAgingRcd.h"
+
+//
+// class declaration
+//
+
+class GEMChamberMasker : public edm::EDProducer 
```

Comment:
This class needs to be a modern producer (probably an `edm::stream::EDProducer`)